### PR TITLE
Completed challenges 0

### PR DESCRIPTION
**Pull request by TharinduDilshan**
I completed challenges 0 and 1

#Hacktoberfest2022

### CODE REVIEW

**Review by matheusth — CHANGES_REQUESTED** [review]
Hi @TharinduDilshan, the format of contribution to this repository,  requires that one pull request solves only one challenge.

**Review by TharinduDilshan — COMMENTED** [review]
Hi [matheusth](https://github.com/matheusth),

I removed the challenge 1. Please review.

File: challenges/0-hello-world/java/tharindusoori/HelloWorld.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+public class HelloWorld{
```
Comment (matheusth) [inline challenges/0-hello-world/java/tharindusoori/HelloWorld.java:1]: Add a space between Helloword and the '{'

File: challenges/0-hello-world/java/tharindusoori/HelloWorld.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    
+	public static void main(String[] args) {
```
Comment (matheusth) [inline challenges/0-hello-world/java/tharindusoori/HelloWorld.java:3]: The block is not indented correctly.

**Review by matheusth — CHANGES_REQUESTED** [review]

File: challenges/0-hello-world/java/tharindusoori/HelloWorld.java (hunk 2 above)
Comment (matheusth) [reply challenges/0-hello-world/java/tharindusoori/HelloWorld.java:3]: The closing was indented with spaces, but the opening of the block was indented with tab, making block appear not aligned.

**Review by matheusth — APPROVED** [review]

**Comment by TharinduDilshan** [comment]
Hi [matheusth](https://github.com/matheusth),

Thanks for approving the changes and could you please add hacktoberfest-accepted label?

**Pull request merged**